rust/ql/test/library-tests/variables/variables.rs

```diff
@@ -432,6 +432,16 @@ fn capture_mut() {
     print_i64(z); // $ read_access=z
 }
 
+async fn async_block_capture() {
+    let mut i: i64 = 0; // i
+    let block = async {
+        i = 1; // $ read_access=i
```

Comment:
Why is this not a `write_access`?

Reply:
Oops. There was even `testFailures` in the `.expected` file that I missed 🙈